test/test_helper.rb

```diff
@@ -16,9 +16,12 @@ def create_user
   def create_shop
     category_1 = Oil.create(name: "Lard")
     category_2 = Oil.create(name: "Coconut Oil")
-    Chip.create(name: "Slotachips", price: 20, description: "Super yummy", oil_id: category_1.id)
-    Chip.create(name: "Dang Coconut", price: 17, description: "Dang, these are good", oil_id: category_2.id)
-    Chip.create(name: "Old Chips", price: 20, description: "Super yummy", oil_id: category_1.id, status: "retired")
+    Chip.create(name: "Slotachips", price: 20,
+    description: "Super yummy", oil_id: category_1.id)
+    Chip.create(name: "Dang Coconut", price: 17,
+    description: "Dang, these are good", oil_id: category_2.id)
+    Chip.create(name: "Old Chips",
+    price: 20, description: "Super yummy", oil_id: category_1.id, status: "retired")
```

Comment:
Line is too long. [84/80]<br>Align the elements of a hash literal if they span more than one line.